src/Substrate.ts

```diff
@@ -93,10 +92,10 @@ export class Substrate {
    *  @throws {Error} when the client encounters an error making the request.
    */
   async runSerialized(
-    serialized: any,
-    nodes: Node[] | null = null,
+    nodes: Node[],
```

Comment:
not really sure why this was optional before but seems like it's not used anywhere else, so it seemed okay to change to required

Reply:
Yeah, not sure why it can also accept `null` there either, but that makes sense to me to remove that since we're always passing it through for now